src/app/main.cpp

```diff
@@ -302,6 +352,38 @@ QBtCommandLineParameters parseCommandLine()
                 result.shouldDaemonize = true;
             }
 #endif
+            else if ((arg == QLatin1String("-p")) || (arg == QLatin1String("--path"))) {
+                if ((i+1) < appArguments.size()) {
+                    result.savePath = appArguments[i+1];
+                    i += 1;
+                }
+                else {
+                    // Invalid arguments
+                    result.errorParameter = arg;
+                }
+            }
+            else if (arg == QLatin1String("--add-started")) {
+                result.addPaused = false;
+            }
+            else if (arg == QLatin1String("--add-paused")) {
```

Comment:
Having both add-started and add-paused params is bad idea. One of them is redundant.